packages/fhir-converter/src/templates/cda/Resources/FamilyMemberHistory.hbs

```diff
@@ -36,7 +36,12 @@
             {{/each}}
         ],
         "status":"completed"
-        "relationship":{{>DataType/CodeableConcept.hbs code=familyEntry.subject.relatedSubject.code}},
+        
+        {{#if familyEntry.subject.relatedSubject.code}}
+          "relationship":{{>DataType/CodeableConcept.hbs code=familyEntry.subject.relatedSubject.code}},
+        {{else}}
+          "relationship":{{>DataType/CodeableConcept.hbs text='n/a'}},
+        {{/if}}
```

Comment:
ditto on updating CodeableConcept itself and not changing template here

Reply:
https://github.com/metriport/metriport/pull/2357#discussion_r1662607267